views/layout.html

```diff
@@ -53,7 +53,7 @@
         Serve a different stylesheet for IE8 :
         http://www.theguardian.com/info/developer-blog/2013/oct/14/mobile-first-responsive-ie8 #}
         <!--[if gt IE 8]><!-->
-        <link rel="stylesheet" href="{{ asset('styles/main.css') }}">
+        <link rel="stylesheet" href="{{ asset('styles/main.css') }}>
```

Comment:
You've got an error here - we need that closing quote otherwise the build will break.